api/src/main/java/org/openmrs/module/registrationcore/RegistrationCoreConstants.java

```diff
@@ -14,29 +14,49 @@
 package org.openmrs.module.registrationcore;
 
 public final class RegistrationCoreConstants {
-	
+
 	/**
 	 * Specifies the identifier source to use when generating patient identifiers
 	 */
 	public static final String GP_IDENTIFIER_SOURCE_ID = "registrationcore.identifierSourceId";
-	
+
 	public static final String DATE_FORMAT_STRING = "yyyy-MM-dd HH:mm:ss";
-	
-	public static final String TOPIC_NAME = "org.openmrs.module.registrationcore.RegistrationEvent";
-	
+
+	public static final String REGISTRATION_EVENT_TOPIC_NAME = "org.openmrs.module.registrationcore.RegistrationEvent";
+
+	public static final String EDIT_EVENT_TOPIC_NAME = "org.openmrs.module.registrationcore.EditEvent";
+
 	public static final String KEY_PATIENT_UUID = "patientUuid";
-	
+
 	public static final String KEY_RELATIONSHIP_UUIDS = "relationshipUuids";
-	
+
 	public static final String KEY_REGISTERER_UUID = "registererUuid";
-	
+
+	public static final String KEY_REGISTERER_ID = "registererId";
+
 	public static final String KEY_DATE_REGISTERED = "dateRegistered";
-	
+
 	public static final String KEY_WAS_A_PERSON = "wasAPerson";
 
     public static final String GP_PATIENT_NAME_SEARCH = "registrationcore.patientNameSearch";
-    
+
     public static final String GP_FAST_SIMILAR_PATIENT_SEARCH_ALGORITHM = "registrationcore.fastSimilarPatientSearchAlgorithm";
-    
-    public static final String GP_PRECISE_SIMILAR_PATIENT_SEARCH_ALGORITHM = "registrationcore.preciseSimilarPatientSearchAlgorithm";
-}
+
+	public static final String GP_PRECISE_SIMILAR_PATIENT_SEARCH_ALGORITHM = "registrationcore.preciseSimilarPatientSearchAlgorithm";
+
+	public static final String GP_MPI_GLOBAL_IDENTIFIER_DOMAIN_ID = "registrationcore.mpi.globalIdentifierDomainId";
+
+	public static final String GP_MPI_PERSON_IDENTIFIER_ID = "registrationcore.mpi.personIdentifierId";
+
+	public static final String GP_LOCAL_MPI_IDENTIFIER_TYPE_MAP = "registrationcore.local_mpi_identifierTypeMap.";
+
+	public static final String GP_MPI_IMPLEMENTATION = "registrationcore.mpi.implementation";
+
+	public static final String GP_MPI_URL = "registrationcore.mpi.url";
+
+	public static final String GP_MPI_ACCESS_USERNAME = "registrationcore.mpi.username";
```

Comment:
Actually, I'm not sure that every MPI implementation necessarily uses username/password. (E.g. it could be token based, or oauth based or something else). So, maybe it's more appropriate for these GPs to be named specific to OpenEMPI.

(But I guess this is fine to leave as-is.)